### ISSUE TITLE
Update Getting Started config.xml editing for iOS

### ISSUE THREAD

**Issue by eerhardt** [Microsoft/cordova-plugin-code-push#59]
Walking through the Getting Started documentation on my iOS device, I was not successful out of the box for the following reasons:

1. `<access origin="https://codepush.azurewebsites.net/" />` didn't work on iOS, I needed to remove the trailing `/`
2. I also needed to add ` <access origin="https://codepush.blob.core.windows.net" />`

**Comment by lostintangent** [comment]
@eerhardt Could you let me know which version of Cordova and Cordova-iOS you're using? We haven't tested the 6.0.0 release that came out on Thursday, so if you're running that, we likely need to tweak our instructions. Otherwise, this would be a new bug to us that I'd love to get more details about the repro. Thanks for reporting this issue!

**Comment by eerhardt** [comment]
I'm a bit behind, which is probably causing most of the problems I've been seeing.

taco.json:

```
{
	"cordova-cli": "5.4.0"
}
```

platforms/platforms.json:

```
{
    "android": "4.0.2",
    "ios": "3.9.2",
    "wp8": "3.8.1",
    "windows": "4.0.0"
}
```

Visual Studio Tools for Apache Cordova: Version 14.0.51016.1.

Does that help?

**Comment by lostintangent** [comment]
Fixed via #62. Thanks for reporting this issue!
